### ISSUE TITLE
Planetscale doesn't make sense, should be on vercel architecture. really easy change with Prisma

### ISSUE THREAD

**Issue by rozenstein94** [shadcn/taxonomy#148]

**Comment by JohnGemstone** [comment]
Why do you think Planetscale doesn't make sense? They offer a great free tier (multiple accounts), backups, branching .etc. Vercel's storage at the moment is pretty bear bones I thought.
If you're going to open an issue you should really detail your thoughts behind this, what pain points you experience.

**Comment by NotellumCreek** [comment]
> Why do you think Planetscale doesn't make sense? They offer a great free tier (multiple accounts), backups, branching .etc. Vercel's storage at the moment is pretty bare bones I thought. If you're going to open an issue you should really detail your thoughts behind this, what pain points you experience.

From Planetscale 

"Note - The Hobby plan will be deprecated on April 8th, 2024, and you are no longer able to create new Hobby databases as of March 6th, 2024. Any Hobby databases that have not been migrated by April 8th, 2024 will be slept. Data will not be deleted, but a paid plan will be required to access data after April 8th. Read the [Hobby plan deprecation FAQ documentation](https://planetscale.com/docs/concepts/hobby-plan-deprecation-faq) more information about next steps."

It was really easy to convert from Planetscale to Vercel PG though. Given that the hobby plan on planetscale is going away, it seems like a good idea to me.

**Comment by JohnGemstone** [comment]
Yeah I've been provisioning different db services all day. Kind of crazy PS havent sent out an email yet (at least I havent received one). Thanks for the heads up 🫡
